src/ExpressionEvaluator/CSharp/Source/ResultProvider/CSharpFormatter.Values.cs

```diff
@@ -136,6 +136,29 @@ internal override string GetCastExpression(string argument, string type, bool pa
             return pooled.ToStringAndFree();
         }
 
+        internal override string GetTupleExpression(string[] values)
```

Comment:
Are the value names included in `values`?  If not is the change to have value names done elsewhere?